### ISSUE TITLE
Feature Request: Add user/team reviewer field for generated PR

### ISSUE THREAD

**Issue by aht007** [saadmk11/github-actions-version-updater#23]
It would be nice to have optional fields for adding team reviewers/user reviewers on the PR that is generated by this action. This way we would also be notified of any newly opened PR due to this action.
I am trying to implement this for our openedX repos here in this PR: https://github.com/openedx/edx-platform/pull/31170 and this feature would unblock me on this PR.

**Comment by saadmk11** [comment]
Hi @aht007, Thanks for creating the issue, I have added this feature here: https://github.com/saadmk11/github-actions-version-updater/pull/24

**Comment by aht007** [comment]
Thanks @saadmk11 for the timely response. Much appreciated

**Comment by aht007** [comment]
Btw We would need a new release for this feature to be available for use right...? But I don't see a new release. Did you forget about it by chance?

**Comment by saadmk11** [comment]
> Btw We would need a new release for this feature to be available for use right...? But I don't see a new release. Did you forget about it by chance?

I am planning to do a new release this weekend. You can test it out by using `saadmk11/github-actions-version-updater@main` as the version.

**Comment by saadmk11** [comment]
Hi @aht007, I have released a new version of this action. https://github.com/saadmk11/github-actions-version-updater/releases/tag/v0.7.0
